pydis_site/apps/content/resources/server-info/roles.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Python Discord Server Roles
+description: Information on the various roles at Python Discord.
+icon: fab fa-discord
+---
+
+# Basic Roles
+
+### <span class="fas fa-circle" style="color:#6e6e6e"></span> Announcements
```

Comment:
I like the change to use circles next to role names rather than colouring the text. This is consistent with what Discord did in their client too.